.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linux:
+    name: "Node ${{ matrix.node }} on Linux: Lint"
```

Comment:
Consequently, I'd rename this to just `Lint` (and maybe the `linux` key above too)

Reply:
Let me do that!

Reply:
[c825542](https://github.com/karma-runner/karma-jasmine/pull/313/commits/c8255429284effe5f74d9538c024ae87054d6cc4)